### PR TITLE
Feat/cards

### PR DESCRIPTION
**Pull request by cnerot**
Gives the merchant the ability to display Card logo as requested by Mdm.

[issue](https://app.clickup.com/t/20427503/MAC-463)

### CODE REVIEW

**Comment by gautierlabarre** [comment]
I have errors in my developper tools 

image

It happens for `mastercard` and `visa` only.

File: src/Widgets/EligibilityModal/DesktopModal/index.tsx [hunk 1]
```diff
@@ -1,16 +1,19 @@
 import cx from 'classnames'
 import React, { FC } from 'react'
+import {Card} from 'types'
```
Comment (ncordin) [inline src/Widgets/EligibilityModal/DesktopModal/index.tsx]:
```suggestion
import { Card } from 'types'
```

Strange Prettier did not catch this

File: microbundle.config.js [hunk 2]
```diff
@@ -29,6 +29,7 @@ module.exports = {
               'src/assets/fonts/PublicSans-VariableFont_wght.ttf',
               'src/assets/fonts/Eina04-Bold.woff',
               'src/assets/fonts/Eina04-Bold.ttf',
+              'src/assets/cards/cb.svg',
```
Comment (ncordin) [inline microbundle.config.js]: I saw you deleted this file, so i think you can remove this line non?

**Review by ncordin — APPROVED** [review]

**Comment by gautierlabarre** [comment]
You don't have any test for this feature

**Comment by gautierlabarre** [comment]
Also I think it'd be nice if we add Static CSS now

File: src/Widgets/EligibilityModal/components/Cards/index.tsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+import MasterCard from 'assets/cards/mastercard'
+import CbCard from 'assets/cards/cb'
+import s from './Cards.module.css'
+type Props = {
+  cards: Card[]
+}
+
+const Cards: FC<Props> = ({ cards }) => {
+  return (
+    <div className={s.cardContainer}>
```
Comment (gautierlabarre) [inline src/Widgets/EligibilityModal/components/Cards/index.tsx]: Also I think it'd be nice if we add Static CSS now.
I think a static class for cardContainer would be enough

File: documentation.md [hunk 4]
```diff
@@ -61,6 +61,10 @@ suggestedPaymentPlan: [10, 4],
 In the above example, the 10 installments plan will be selected. If it's not eligible, the 4 installments plan will be selected. If the 4 installments plan is not eligible, it will select the first tab.
 When `suggestedPaymentPlan` is used, the transition is disable. Unless `transitionDelay` is specified.
 
+- cards: ('cb' | 'visa' | 'amex' | 'mastercard')[] [optional]
```
Comment (gautierlabarre) [inline documentation.md:64]: You might need to add this to the modal documentation as well

**Review by gautierlabarre — APPROVED** [review]

**Pull request merged**